### ISSUE TITLE
Glean dictionary: should we show the `category.name` format instead of `category_name` transformed format?

### ISSUE THREAD

**Issue by chutten** [mozilla/glean-dictionary#242]
Or maybe we should show both? (in the details view perhaps)? I find categories helpful in understanding the organization of metrics, so it would be nice to be able to more easily pick them out.

**Comment by wlach** [comment]
@chutten I have to confess I'm not sure what you mean here. Can you take a look at https://glean-dictionary-dev.netlify.app and give me an example of what you'd like changed?

**Comment by chutten** [comment]
For example, `storage.stats.app_bytes`: https://glean-dictionary-dev.netlify.app/#!/apps/fenix/metrics/storage.stats.app_bytes

Looks like you're using the name as described in the yaml whereas the name present in datasets would be `storage_stats_app_bytes`. Being able to search using either punctuation could be helpful.

**Comment by wlach** [comment]
Yeah, possibly a set of direct links to their representation in the schema would be useful. e.g. in this case:

https://glean-dictionary-dev.netlify.app/#!/apps/fenix/tables/metrics?search=metrics.memory_distribution.storage_stats_app_bytes
